### PR TITLE
add Chinese version of init_cfg

### PR DESCRIPTION
**Pull request by yarkable**
Translate the init_cfg.md doc to Chinese

### CODE REVIEW

File: docs/zh_cn/tutorials/init_cfg.md [hunk 1]
```diff
@@ -0,0 +1,167 @@
+# 教程 10: 权重初始化
+
+
+
+在训练过程中，适当的初始化策略有利于加快训练速度或获得更⾼的性能。 [MMCV](https://github.com/open-mmlab/mmcv/blob/master/mmcv/cnn/utils/weight_init.py) 提供了一些常⽤的初始化模块的⽅法，如 `nn.Conv2d`。 MMdetection 中的模型初始化主要使⽤ `init_cfg`。⽤⼾可以通过以下两个步骤来初始化模型：
+
+1. 在 `model_cfg` 中为模型或其组件定义 `init_cfg`，但⼦组件的 `init_cfg` 优先级更⾼，会覆盖⽗模块的 `init_cfg` 。
+2. 像往常一样构建模型，但显式调⽤ `model.init_weights()` ⽅法，并且模型参数将被初始化为配置。
```
Comment (hhaAndroid) [inline docs/zh_cn/tutorials/init_cfg.md:8]:
```suggestion
2. 像往常一样构建模型，然后显式调⽤ `model.init_weights()` ⽅法，此时模型参数将会被按照配置文件写法进行初始化。
```

File: docs/zh_cn/tutorials/init_cfg.md [hunk 2]
```diff
@@ -0,0 +1,167 @@
...
+
+
+
+MMdetection 中初始化的⾼级⼯作流程是：
```
Comment (hhaAndroid) [inline docs/zh_cn/tutorials/init_cfg.md:12]: MMdetection 初始化工作流的高层 API 调用流程是：

**Comment by hhaAndroid** [comment]
@yarkable Please fix the lint first

**Review by zytx121 — CHANGES_REQUESTED** [review]
Error operation, please ignore.

**Comment by zytx121** [comment]
Hi @yarkable  
You can use pre-commit to fix lint errors by run `pre-commit run --all-files`

**Review by Czm369 — APPROVED** [review]
LGTM

File: docs/zh_cn/tutorials/init_cfg.md [hunk 3]
```diff
@@ -0,0 +1,167 @@
...
+
+### 描述
+
+
+
+它的数据类型是 dict 或者 list[dict]，包含了下列键值:
+
+- `type` (str)，包含 `INTIALIZERS` 中的初始化程序名称，后面跟着初始化程序的参数。
+- `layer`（str 或 list[str]），包含 Pytorch 或 MMCV 中基本层的名称，以及将被初始化的可学习参数，例如 `'Conv2d'`，`'DeformConv2d'`。
+- `override` (dict or list[dict])，包含不继承⾃ `BaseModule` 且其初始化配置与 `layer` 键中的其他层不同的⼦模块。 `type` 中定义的初始化器将适⽤于 `layer` 中定义的所有层，因此如果⼦模块不是 `BaseModule` 的派⽣类但可以与 `layer` 中的层相同的⽅式初始化，则不需要使⽤ `override`。`override` 包含了：
```
Comment (zytx121) [inline docs/zh_cn/tutorials/init_cfg.md:24]: - `override` (dict 或 list[dict])

File: docs/zh_cn/tutorials/init_cfg.md [hunk 4]
```diff
@@ -0,0 +1,167 @@
...
+
+model_cfg(init_cfg) -> build_from_cfg -> model -> init_weight() -> initialize(self, self.init_cfg) -> children's init_weight()
+
+### 描述
+
+
+
+它的数据类型是 dict 或者 list[dict]，包含了下列键值:
+
+- `type` (str)，包含 `INTIALIZERS` 中的初始化程序名称，后面跟着初始化程序的参数。
```
Comment (zytx121) [inline docs/zh_cn/tutorials/init_cfg.md:22]: - `type` (str)，包含 `INTIALIZERS` 中的初始化器的名称，以及后面跟着的初始化器的参数。

File: docs/zh_cn/tutorials/init_cfg.md [hunk 5]
```diff
@@ -0,0 +1,167 @@
...
+  # self.cls = nn.Linear(1,2)
+  
+  init_cfg = dict(type='Constant',
+                  layer=['Conv1d','Conv2d'], val=1, bias=2,
+                  override=dict(type='Constant', name='reg', val=3, bias=4))
+  # self.feat and self.cls 将被初始化为 dict(type='Constant', val=1, bias=2)
+  # 叫 'reg' 的模块将被初始化为 dict(type='Constant', val=3, bias=4)
+  ```
+
+- 如果 init_cfg 中的 `layer` 为 None，则只会初始化 override 中名称的⼦模块，⽽ override 中的 type 和其他参数可以省略。
```
Comment (zytx121) [inline docs/zh_cn/tutorials/init_cfg.md:132]: - 如果 init_cfg 中的 `layer` 为 None，则只会初始化 override 中有名称的⼦模块，⽽ override 中的 type 和其他参数可以省略。

**Comment by yarkable** [comment]
pre-commit run --all-files

File: docs/zh_cn/tutorials/init_cfg.md [hunk 6]
```diff
@@ -0,0 +1,167 @@
+# 教程 10: 权重初始化
+
+
```
[the remark this answers is not in the document]
Comment (yarkable) [reply docs/zh_cn/tutorials/init_cfg.md:3]: OK

**Comment by yarkable** [comment]
pre-commit run --all-files

**Comment by zytx121** [comment]
Hi @yarkable 
Please refer to https://github.com/open-mmlab/mmcv/blob/master/CONTRIBUTING.md for proper use of pre-commit.

**Comment by yarkable** [comment]
> 

Thanks a lot! It's all passed now.

```bash
[WARNING] The 'rev' field of repo 'https://github.com/open-mmlab/pre-commit-hooks' appears to be a mutable reference (moving tag / branch).  Mutable references are never updated after first install and are not supported.  See https://pre-commit.com/#using-the-latest-version-for-a-repository for more details.  Hint: `pre-commit autoupdate` often fixes this.
flake8...................................................................Passed
seed isort known_third_party.............................................Passed
isort....................................................................Passed
yapf.....................................................................Passed
Trim Trailing Whitespace.................................................Passed
Check Yaml...............................................................Passed
Fix End of Files.........................................................Passed
Fix requirements.txt.....................................................Passed
Fix double quoted strings................................................Passed
Check for merge conflicts................................................Passed
Fix python encoding pragma...............................................Passed
Mixed line ending........................................................Passed
Markdownlint.............................................................Passed
codespell................................................................Passed
docformatter.............................................................Passed
check algorithm readme...................................................Passed
check copyright..........................................................Passed
```

**Review by zytx121 — APPROVED** [review]

**Comment by zytx121** [comment]
LGTM.

File: docs/zh_cn/tutorials/init_cfg.md [hunk 7]
```diff
@@ -0,0 +1,162 @@
...
+  # 叫 'reg' 的模块将被 dict(type='Constant', val=1, bias=2) 初始化
+  ```
+
+- 如果我们不定义 `layer` 或 `override` 键，它不会初始化任何东西。
+
+- 无效的使用
+
+  ```python
+  # override 没有 name 键的话是无效的
+  init_cfg = dict(type='Constant', layer ['Conv1d','Conv2d'], val=1, bias=2,
```
Comment (BIGWangYuDong) [inline docs/zh_cn/tutorials/init_cfg.md:147]: miss `=` here : layer=['Conv1d','Conv2d']

File: docs/zh_cn/tutorials/init_cfg.md [hunk 8]
```diff
@@ -0,0 +1,162 @@
...
+              	override=dict(type='Constant', val=3, bias=4))
+
+  # override 有 name 键和其他参数但是没有 type 键也是无效的
+  init_cfg = dict(type='Constant', layer ['Conv1d','Conv2d'], val=1, bias=2,
```
Comment (BIGWangYuDong) [inline docs/zh_cn/tutorials/init_cfg.md:151]: layer=['Conv1d','Conv2d']
Also, please fix this error in English version

**Comment by BIGWangYuDong** [comment]
@yarkable Please remember to add `=` in the English version document at the same time.

**Comment by yarkable** [comment]
> @yarkable Please remember to add `=` in the English version document at the same time.

Fixed now.
